### PR TITLE
Expand catalog index rebuild documentation

### PR DESCRIPTION
**Pull request by aelkiss**
@moseshll  you mentioned you ran into a couple of permissions issues which you thought might be worth documenting

### CODE REVIEW

**Review by moseshll — CHANGES_REQUESTED** [review]
Recommend the small set of changes I suggested in PR #25 be merged into this doc. Or, I can do a commit on this branch if you like.

File: solr/README.md [hunk 1]
```diff
@@ -5,27 +5,66 @@ All indexing is done on the "reindexing solr", currently
 every night to the production servers, which we never actually need to 
 touch.
 
-Deploying a new solr configuration is as follows:
+Deploying a new solr configuration is as follows. First, get the new code:
 
 ```shell
 cd /htsolr/catalog/bin/ht_catalog_indexer
-git pull origin main # get the new code
+# verify there is nothing uncommitted with git status; commit or git stash as needed
```
Comment (billdueber) [inline solr/README.md:12]: "It's expected that the translation maps (e.g., `collection_code_to_original_from.yaml`) will have changed, as they're generated from the database on each run."

**Review by moseshll — APPROVED** [review]
I've added my suggestions to this PR and closed the branch/PR I made independently.

File: solr/README.md (hunk 1 above)
Comment (aelkiss) [reply solr/README.md]: Added a note.

**Pull request merged**